python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -414,7 +427,7 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgInfoL
         for subBBox in _subBBoxIter(skyInfo.bbox, subregionSize):
             try:
                 self.assembleSubregion(coaddExposure, subBBox, tempExpRefList, imageScalerList,
-                                       weightList, bgInfoList, statsFlags, statsCtrl)
+                                       weightList, bgInfoList, altMaskList, statsFlags, statsCtrl)
             except Exception, e:
```

Comment:
This was not changed, but since we are editing this file can we please update the except here to be:
```python
except Exception as e:
```
A distinct commit is fine and there are 5 in this file.